frontend-beeyond/src/app/core/header/header.component.scss

```diff
@@ -17,3 +17,14 @@
 .header-title {
   margin-left: 22px;
 }
+
+a[ng-reflect-router-link] {
```

Comment:
Maybe use classes instead of these attributes. Not everbody knows the internal attributes of angular